### PR TITLE
Okta-Access-Gateway - (few changes)

### PR DESCRIPTION
**Pull request by samiksha2005**
Added new stream - THREAT 
Added 3 display msg in mapping - Verify user identity , Send second factor auth SMS , Request from suspicious actor

### CODE REVIEW

File: extractors/ACCESS-GATEWAY/okta.yaml [hunk 1]
```diff
@@ -317,9 +318,47 @@ event-details:
...
       translate:
         actor[displayName]: User
-        ipAddress: SrcIP
+        client[ipAddress]: SrcIP
         AuthProto: AuthProto 
         outcome[reason]: Reason
+    "Send second factor auth SMS-SUCCESS":
+      annotate:
+        Stream: AUTHENTICATION
+        Action: LOGIN
```
Comment (Sherin18) [inline extractors/ACCESS-GATEWAY/okta.yaml:327]: Take Action as Token_Sent .

File: extractors/ACCESS-GATEWAY/okta.yaml [hunk 2]
```diff
@@ -317,9 +318,47 @@ event-details:
...
+        Status: PASSED
+      translate:
+        actor[displayName]: User
+        client[ipAddress]: SrcIP
+        AuthProto: AuthProto
+    "Send second factor auth SMS-FAILURE":
+      annotate:
+        Stream: AUTHENTICATION
+        Action: LOGIN
```
Comment (Sherin18) [inline extractors/ACCESS-GATEWAY/okta.yaml:336]: Action =Token_Sent

File: extractors/ACCESS-GATEWAY/okta.yaml (hunk 1 above)
Comment (samiksha2005) [reply extractors/ACCESS-GATEWAY/okta.yaml:327]: I'll replaced action login with 2FA_TOKEN_SENT

File: extractors/ACCESS-GATEWAY/okta.yaml (hunk 2 above)
Comment (samiksha2005) [reply extractors/ACCESS-GATEWAY/okta.yaml:336]: i will added  2FA _tOKEN_SENT

**Comment by GayatriBatteli** [comment]
Validated Stream: AUTHENTICATION, IAM, THREAT, CONFIGURATION

**Pull request merged**